README.md

```diff
@@ -122,8 +122,35 @@ if (process.env.NODE_ENV === 'production') {
 }
 ```
 
+### env.disableError()
```

Comment:
Could you call this `disableErrors()` so that the wording is consistent with `disableFallbacks()`?